test/unit/modules/blocks/utils.js

```diff
@@ -121,6 +113,14 @@ describe('blocks/utils', () => {
 					expect(library.logic.transaction).to.eql(transactionMock);
 				});
 			});
+
+			it('should assign dbSequence', () => {
+				expect(library.dbSequence).to.eql(modulesLoader.scope.dbSequence);
+			});
+
+			it('should assign db', () => {
+				expect(library.db).to.eql(dbStub);
+			});
```

Comment:
I was following the order in which each parameter is assigned to `library` object, not the order of invocation in the function constructor, because some functions have `scope` as input parameter, therefore it can not be the standard.